pages/common/git-add.md

```diff
@@ -17,3 +17,7 @@
 - Also add ignored files:
 
 `git add -f`
+
+- Add parts of a file interactively
+
+`git add -p {{PATHSPEC}}`
```

Comment:
I like @ostera's version more, I don't expect many people would try adding files from outside the project directory in order to justify @igorshubovych's caveat.